pybossa/cache/users.py

```diff
@@ -366,9 +367,24 @@ def delete_taskbrowse_bookmarks(user):
     delete_memoized(get_user_by_id, user.id)
 
 
-def get_user_preferences(user_id):
+def get_user_preferences(user_id, map_to_country_codes=False):
```

Comment:
It would also be useful to include unit test(s) to cover the new logic related to `location` being a country code versus name.

Reply:
added test cases